### PR TITLE
[Refactor] Remove OSReleaseFacts

### PR DESCRIPTION
**Pull request by drehak**
Functionality of the model `OSReleaseFacts` pretty much got moved into `OSRelease`, a part of `IPUConfig`.
This actor removes all uses of the former and replaces them with the latter where possible.

### CODE REVIEW

File: repos/system_upgrade/el7toel8/libraries/config/version.py [hunk 1]
```diff
@@ -137,15 +137,15 @@ def current_version():
     :return: The tuple contains release name and version value.
     :rtype: (string, string)
     """
-    facts_messages = api.consume(OSReleaseFacts)
+    facts_messages = api.consume(IPUConfig)
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/libraries/config/version.py]: Everyactor has the configuration included. You are not supposed to consume it. `api.current_actor().configuration.os_release`. From this point, the check doesn't make sense probably so much as well. As well, the original lib is wrong probably as it cannot use consume without check that actor is consuming the msg. As well, Import is not needed in such case.

**Comment by drehak** [comment]
leapp-ci build

File: repos/system_upgrade/el7toel8/libraries/config/version.py (hunk 1 above)
Comment (drehak) [reply repos/system_upgrade/el7toel8/libraries/config/version.py]: ok

**Comment by examon** [comment]
leapp-ci build

**Comment by drehak** [comment]
leapp-ci build

**Comment by examon** [comment]
leapp-ci build

**Comment by fellipeh** [comment]
e2e tests

**Comment by jmikovic** [comment]
please rebase

**Comment by fellipeh** [comment]
@drehak please rebase and squash.

**Comment by fellipeh** [comment]
e2e tests

**Comment by drehak** [comment]
leapp-ci build

**Comment by pirat89** [comment]
lol. the new notification system showed me this one as opened :D

**Pull request closed without merging**